### DISCUSSION TITLE
add any url to whitelist

### DISCUSSION THREAD

**Discussion by irfanonk** [Web3Auth/Web3Auth#665] [Help]
i would like to add any url that ends with "vercel.app" to whitelist. so it could validate redirect url after each deployment to vercel for vercel's unique url. Is there any way to achieve it.

**Comment by shahbaz17** [comment]
Hey @irfanonk 

Currently, there's no way to whitelist `*.vercel.app` on the Web3Auth dashboard. But please check out [this answer](https://github.com/orgs/Web3Auth/discussions/351?#discussioncomment-3303734) and see if this helps.
> Note: here we are using Client Secret, and it can't be exposed.

**Comment by himanshuchawla009** [comment]
Hey @irfanonk , refer to the answer linked below that might help you in fulfilling your usecase.

https://github.com/orgs/Web3Auth/discussions/398
